rpcs3/Loader/PSF.cpp

```diff
@@ -21,6 +21,23 @@ void fmt_class_string<psf::format>::format(std::string& out, u64 arg)
 	});
 }
 
+template<>
+void fmt_class_string<psf::error>::format(std::string& out, u64 arg)
+{
+	format_enum(out, arg, [](auto fmt)
+	{
+		switch (fmt)
+		{
+		case psf::error::stream: return "File is non-existant";
```

Comment:
Typo: non-exist __e__ nt

Reply:
or just say "File does not exist"

Reply:
"File doesn't exist" would be better english.